apps/prairielearn/src/lib/format.test.ts

```diff
@@ -0,0 +1,58 @@
+import { assert } from 'chai';
+
+import { formatFloat, formatPoints, formatPointsOrList } from './format.js';
+
+describe('format library', () => {
+  it('formatFloat', () => {
```

Comment:
Total nit: typically `describe` would be the name of the thing under test (e.g. `formatFloat`) and `it` would be a description of the test, e.g. something that reads like `it('formats with two decimal digits by default')` and so on.